packages/common/src/converter/payload-converter.ts

```diff
@@ -100,6 +100,25 @@ export function mapFromPayloads<K extends string, T = unknown>(
   ) as Record<K, T>;
 }
 
+/**
+ * RawValue is a wrapper over a payload.
+ * A payload that belongs to a RawValue is special in that it bypasses user-defined payload converters,
+ * instead using the default payload converter. The payload still undergoes codec conversion.
+ */
+export declare const rawPayloadTypeBrand: unique symbol;
```

Comment:
The typedoc must be _immediately_ followed by the `export class` line. As it is now, the typedoc attaches to the `rawPayloadTypeBrand` const.